collections/_article/platform-state-in-csharp-for-godot-4-2.md

```diff
@@ -0,0 +1,99 @@
+
+Before **.NET unification** there was the Windows-only .NET Framework and the cross-platform Mono and .NET Core. Unification means there will be just one .NET going forward, so the next release after .NET Core 3.0 was named .NET 5, and Mono and .NET Framework won't have any new major releases.
+
+The term **Mono** can be used to refer to a collection of technologies. With .NET unification, the Mono framework is deprecated, but the runtime is still supported. Unified .NET uses both the CoreCLR and the Mono runtimes, but Mono is the only runtime that supports mobile and web platforms.
+
+However, in .NET 7.0 a new runtime became available, [**NativeAOT**](https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot), which allows publishing native binaries built for a specific platform, instead of the JIT-based portable binaries. In .NET 7.0, NativeAOT only supports Windows and Linux, but in .NET 8.0 it adds support for MacOS and experimental support for Android and iOS. This means in .NET 8.0 NativeAOT can be used as an alternative to Mono for mobile platforms.
+
+## Platform support in 4.2
+
+Thanks to the amazing work by [RedworkDE](https://github.com/RedworkDE) and [Andreia Gaita](https://github.com/shana), C# projects in Godot [4.2](/article/godot-4-2-arrives-in-style) have _experimental_ support for exporting to **Android** and **iOS**.
+
+This initial implementation has some limitations which is why they are still marked experimental, we want to make sure the platform support is production-ready when we remove the experimental label. The platform limitations are documented in the [C# documentation](https://docs.godotengine.org/en/4.2/tutorials/scripting/c_sharp/index.html#c-platform-support), but we'll go over each platform here.
+
+### Android
+
+Android support was implemented in [GH-73257](https://github.com/godotengine/godot/pull/73257) by [RedworkDE](https://github.com/RedworkDE) and uses the .NET hosting APIs provided, just like the desktop platforms.
+
+Exporting to Android requires .NET **7.0** or higher, uses the [**linux-bionic**](https://github.com/dotnet/runtime/pull/66147) Mono runtime, and only supports the `arm64` and `x64` architectures.
+
+The **linux-bionic** runtime is a linux runtime using the Android C library (so it's basically Android but without the JNI). This means that the Android bindings are not available, so some APIs (such as [SSL](https://github.com/godotengine/godot/issues/84559)) will crash the game, use the Godot APIs when possible to avoid these issues.
```

Comment:
```suggestion
The **linux-bionic** runtime is a Linux runtime using the Android C library, so it's basically Android but without the JNI. This means that the Android bindings are not available, so some APIs (such as [SSL](https://github.com/godotengine/godot/issues/84559)) will crash the game. Use the Godot APIs when possible to avoid these issues.
```